kaltura/playkit-android
# FEM-2085: Upgrade ExoPlayer to v2.8.2

### Description of the Changes
CustomVideoCodecRenderer.java:
+ Update with latest exoplayer implementation

SimpleDashParser.java:
+ remove deprecated check for LoadCanceled (now implemented inside exoplayer)

ExoPlayerWrapper.java:
+ Fix deprecated methods.
+ DefaultBandwidthMeter now use Builder pattern.
+ Exoplayer now handles TrackSelection.Factory by itself so there is no need to pass it and manage in TrackSelectionHelper.java

TrackSelectionHelper.java:
+ Remove unused FixedTrackSelection.Factory(exoplayer now handles it by itself)
+ Remove unused Constants
+ Some small performance improvement. No need to use StringBuilder when create uniqueId. Concatenation can be done with String.
+ Some small refactor for better code readability.
+ Use new approach for work with Exoplayer`s DefaultTrackSelector.
DefaultTrackSelector.ParametersBuilder should be modified when we want to change TrackSelector values like enable/disable renderer, making overrides e.t.c.
